.github/workflows/env-publish-test-base-image.yaml

```diff
@@ -0,0 +1,27 @@
+name: (k8s package) Publish Test Base Image
+on:
+  push:
+    tags:
+      - 'v*'
+    paths:
+    - "k8s/**"
+
+jobs:
+  publish_test_base_image:
+    if: ${{ false }}
```

Comment:
Is this still needed?

Reply:
nope! and that reminds that I have to restore old triggers for `chainlink-env` specific jobs